playbooks/roles/load-balancer-v2/templates/haproxy.cfg.ctmpl

```diff
@@ -119,6 +119,7 @@ backend provisioning
   {{- with $config := .Value | parseJSON }}
     {{- if $config }}
     {{ $health_check := $config.health_checks_enabled }}
+    {{- if $config.active_app_servers }}
```

Comment:
@shimulch, this condition can be combined into the existing if condition with `and` ([docs](https://golang.org/pkg/text/template/#hdr-Functions))

Reply:
@lgp171188 Thanks. Updated.